cmd/fluxd/main.go

```diff
@@ -134,6 +133,8 @@ func main() {
 			logger.Log("err", err)
 			os.Exit(1)
 		}
+		clientset.Core()
```

Comment:
Ha, OK. I wondered what it did!

> aside: How on earth does this not error!

Because the return value(s), error or otherwise, are discarded.